main.go

```diff
@@ -154,7 +154,7 @@ func mainE(ctx context.Context) error {
 	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.CrtFile, "", "Certificate file path to use to authenticate with Kubernetes.")
 	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.KeyFile, "", "Key file path to use to authenticate with Kubernetes.")
 
-	daemonCommand.PersistentFlags().String(f.Service.RegistryDomain, "quay.io", "Image registry.")
+	daemonCommand.PersistentFlags().String(f.Service.RegistryDomain, "", "Image registry. Empty value will cause using Docker Hub.")
```

Comment:
I wouldn't add this assumption here. The default may change and nobody will fix this here.

Reply:
We use the configuration from ConfigMap so I think we are good here. I just wanted to remove quay reference as we are going to stop using it.